app/jobs/message_relay_job.rb

```diff
@@ -4,12 +4,7 @@ class MessageRelayJob < ApplicationJob
   def perform(message)
     ActionCable
       .server
-      .broadcast "messages_for_#{message.room_id}", message: {
-        id: message.id,
-        content: message.content,
-        name: message.user.name,
-        created_at: message.created_at,
-        room_id: message.room_id
-      }
+      .broadcast "messages_for_#{message.room_id}",
```

Comment:
just curious, what are the security implications of this broadcast? can anyone knowing the room id subscribe and receive messages?